go-controller/Makefile

```
@@ -92,21 +92,26 @@ clean:
+
 lint:
 ifeq ($(CONTAINER_RUNNABLE), 0)
-	@GOPATH=${GOPATH} ./hack/lint.sh $(CONTAINER_RUNTIME) || { echo "lint failed! Try running 'make lint-fix'"; exit 1; }
+	@GOPATH=${GOPATH} $(LINT_SH) $(CONTAINER_RUNTIME) || { echo "lint failed! Try running 'make lint-fix'"; exit 1; }
 else
 	echo "no container runtime. attempting to run natively";
-	@GOPATH=${GOPATH} ./hack/lint.sh run-natively || { echo "running lint locally failed!"; exit 1; }
+	@GOPATH=${GOPATH} $(LINT_SH) run-natively || { echo "running lint locally failed!"; exit 1; }
 endif
 
 lint-fix:
```

checkmake (0.3.2)

[warning] Target "lint-fix" should be declared PHONY.

(phonydeclared)